wikihow2zim/scraper.py

```diff
@@ -29,9 +38,73 @@ def __init__(self, **kwargs):
         for option in options:
             setattr(self, option, kwargs[option])
 
+        if self.output_dir is None:
+            self.output_dir = pathlib.Path(os.getcwd()).joinpath("output")
+
+        pathlib.Path(self.output_dir).mkdir(parents=True, exist_ok=True)
+
         self.url = URLS[self.language]
 
+        # jinja2 environment setup
+        self.env = Environment(
+            loader=FileSystemLoader(str(self.templates_dir)), autoescape=True
+        )
+
+        # Set up the output zim file:
+        fpath = pathlib.Path(self.output_dir).joinpath("output.zim")
+        self.creator = Creator(filename=fpath).set_mainpath("Home")
+        self.creator.start()
+
+    @property
+    def templates_dir(self):
+        return ROOT_DIR.joinpath("templates")
+
+    def add_assets(self):
+        assets_path = ROOT_DIR.joinpath("assets")
+        for root, dirs, files in os.walk(assets_path):
```

Comment:
`dirs` is not used. Don't declare it.
We also prefer using pathlib's helpers rather than `os` ones, when possible.
You may also want to add a debug print

```py
assets_root = ROOT_DIR.joinpath("assets")
for fpath in assets_root.glob("**/*"):
    if not fpath.is_file():
        continue
    logger.debug(f"Assed {fpath.relative_to(assets_root)}")
    self.creator.add_item_for(path=str(fpath.relative_to(assets_root)), fpath=fpath)
````